infrastructure/tooling/src/changelog/index.js

```diff
@@ -129,10 +153,134 @@ class ChangeLog {
   }
 }
 
-const main = async (options) => {
+const check_pr = async (pr) => {
+  const files = [];
+  let url = `https://api.github.com/repos/taskcluster/taskcluster/pulls/${pr}/files`;
+  while (1) {
+    const res = await request
+      .get(url)
+      .set('user-agent', 'taskcluster-ci');
+    res.body.forEach(({filename}) => files.push(filename));
+    const link = res.get('link');
+    if (link) {
+      const parsed = parseLinkHeader(link);
+      if (parsed.next) {
+        url = parsed.next.url;
+        console.log(url);
```

Comment:
stray log.

Reply:
would all of this be simpler with the gihub client? If not, this is fine

Reply:
Good point, the pagination might be easier.